spaceR/spaceR/ViewModels/RocketViewModel.swift

```diff
@@ -0,0 +1,43 @@
+//
+//  RocketViewModel.swift
+//  spaceR
+//
+//  Created by Kieran Woodrow on 2022/03/08.
+//
+
+import Foundation
+
+class RocketViewModel{
+    
+    private var allRockets: [Rocket]
+    
+    init() {
+        self.allRockets = []
+    }
+    
+    func setAllRockets(rockets: [Rocket]) {
+        allRockets = rockets
+    }
+    
+    func getAllRockets() -> [Rocket] {
+        return allRockets
+    }
+    
+    func getRocketCount() -> Int {
+        return allRockets.count
+    }
+    
+    func getRocketImage(index: Int) -> String {
+        if index == 3 {
+            return allRockets[index].flickr_images[1] ?? ""
+        }
```

Comment:
So we like to keep our if statements nicely formatted: 

```
if index == 3 {
   return allRockets[index].flickr_images[1] ?? ""
} else {
   return allRockets[index].flickr_images[0] ?? ""
}
```

Is there a reason that you are specifically checking for `3`?  Normally hard coding things like this is a little but of a warning sign? If they were to add another object to the API then this may not behave as expected?

Reply:
Yeah, i didn't like the image for the 3rd card that is being returned at index 0 of the images array..I didn't know how to fix that so i just left it as a "quick fix" for now